packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -456,6 +471,7 @@ SwipeableDrawer.defaultProps = {
   disableSwipeToOpen:
     typeof navigator !== 'undefined' && /iPad|iPhone|iPod/.test(navigator.userAgent),
   swipeAreaWidth: 20,
+  hysteresis: 0.1,
```

Comment:
This should be `0.5` by default, otherwise it would be a breaking change.

Reply:
indeed, thats true